src/coins.c

```diff
@@ -2,32 +2,38 @@
 
 #include "global.h"
 
-void InitCoins(u16 *coins) {
+void Coins_Init(u16 *coins)
+{
```

Comment:
Please revert brace style
```suggestion
void Coins_Init(u16 *coins) {
```